python/dgl/transform.py

```diff
@@ -221,8 +221,6 @@ def to_bidirected(g, readonly=None, copy_ndata=False):
                 "unidirectional bipartite graphs" \
                 ", but {} is unidirectional bipartite".format(c_etype)
 
-    assert g.is_multigraph is False, "to_bidirected only support simple graph"
```

Comment:
User should make sure the graph is a single graph.
Here we check the single graph.

Reply:
i understand. however the flag is True by default. If I load an OGB graph, its flag is true. When the flag is true, it doesn't mean this is a multigraph.